.changeset/empty-lemons-ring.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/abi-typegen": minor
+"@fuel-ts/contract": minor
+"create-fuels": minor
+---
+
+feat!: prettify `typegen` api
```

Comment:
I believe we need to include `account` here as well

Reply:
and `fuels`